app/views/trainees/funding/bursaries/_non_tiered_bursary_form.html.erb

```diff
@@ -19,20 +19,32 @@
     ) %>
   </p>
 
-  <%= f.govuk_radio_buttons_fieldset(:applying_for_bursary, legend: { text: t("views.forms.funding.bursaries.title"), size: "m" }) do %>
+  <%= f.govuk_radio_buttons_fieldset(:funding_type, legend: { text: t("views.forms.funding.bursaries.title"), size: "m" }) do %>
       <%= f.govuk_radio_button(
-        :applying_for_bursary,
-        "true",
-        label: { text: t("views.forms.funding.bursaries.true") },
+        :funding_type,
+        f.object.class::BURSARY_TYPE,
+        label: { text: t("views.forms.funding.funding_type.bursary.label") },
         link_errors: true,
       ) %>
 
+    <div class="govuk-radios__divider">or</div>
```

Comment:
This should be within the `if` statement below - otherwise we now have an extra 'or':
<img width="578" alt="Screenshot 2021-09-17 at 11 39 17" src="https://user-images.githubusercontent.com/18436946/133769668-618c7439-6754-4d6a-91ff-33ef6c71ce86.png">